### ISSUE TITLE
CAPABILITY_AUTO_EXPAND required for SAM templates

### ISSUE THREAD

**Issue by arcturial** [cloudreach/sceptre#538]
I am trying to create a stack using SAM templates like so:

```
Resources:
  WebhookApi:
    Type: AWS::Serverless::API
    Properties:
      StageName: !Ref StageName
      DefinitionUri: swagger.yml
```
I receive the following message when running it.

`An error occurred (InsufficientCapabilitiesException) when calling the CreateStack operation: Requires capabilities : [CAPABILITY_AUTO_EXPAND]`

Are SAM templates not fully supported?

**Comment by fabiodouek** [comment]
Hi @arcturial , I'm having the same issue. Did you find a solution for this?

**Comment by kkessels** [comment]
From what I can see this is currently not possible because in order to perform the transform we need the CAPABILITY_AUTO_EXPAND capability. Currently, when Sceptre deploys it only provides the other two capabilities: CAPABILITY_IAM and CAPABILITY_NAMED_IAM. I did a small test by adding the CAPABILITY_AUTO_EXPAND to the plan actions and this allowed me to deploy SAM templates using Sceptre.

**Comment by adv4000** [comment]
I fixed it in Sceptre V1.4 by editing **/usr/local/lib/python2.7/dist-packages/sceptre/stack.py**
and added **CAPABILITY_AUTO_EXPAND** 
Lines: 196,235,527 need to be changed 
From: **"Capabilities": ['CAPABILITY_IAM', 'CAPABILITY_NAMED_IAM'],**
To:  **"Capabilities": ['CAPABILITY_IAM', 'CAPABILITY_NAMED_IAM', 'CAPABILITY_AUTO_EXPAND'],**
